src/components/developer-logo/developer-logo.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 import logo from "./assets/logo.png";
 
 import "./developer-logo.scss";
 
-export const DeveloperLogo = () => {
+export const DeveloperLogo = ({ src = null }) => {
```

Comment:
Unless I'm missing something, the `null` default is redundant. `undefined` will still evaluate as falsey